marlowe-dashboard-client/src/Toast/State.purs

```diff
@@ -39,18 +60,21 @@ toastTimeoutSubscription toast =
 handleAction
   :: forall m slots msg
    . MonadAff m
+  => MonadStore Store.Action Store.Store m
   => Action
   -> HalogenM State Action slots msg m Unit
-handleAction (AddToast toast) = do
-  timeoutSubscription <- subscribe $ toastTimeoutSubscription toast
-  assign _mToast (Just { message: toast, expanded: false, timeoutSubscription })
+handleAction (Receive mToast) = do
+  H.put $ deriveState mToast
+  sub <- traverse (subscribe <<< toastTimeoutSubscription) mToast
+  H.modify_ _ { timeoutSubscription = sub }
```

Comment:
Is it a danger that we overwrite a previous subscription here? Do we need to check and unsubscribe or it cannot happen?